### PR TITLE
Fixed white space and word overwrap via mobile

### PR DESCRIPTION
**Pull request by yglazko**
Two-line fix, release notes no longer overflow and force the page to create a white space as they once did.

### CODE REVIEW

File: media/css/firefox/releasenotes.less [hunk 1]
```diff
@@ -447,6 +447,8 @@ article {
     .section-items {
         > li {
             padding-left: 0;
+            word-wrap:break-word;
+            width:50%;
```
Comment (jpetto) [inline media/css/firefox/releasenotes.less]: I think the `word-wrap` property is all we need here? Setting the `width` to 50% pushes text a bit close to the right edge, and results in some overly skinny text on `<li>` elements without an icon - like the "Reference" line on the Android release notes:

screen shot 2015-08-03 at 10 09 01 am

Comment (craigcook) [inline media/css/firefox/releasenotes.less]: I'd take it further and suggest removing the left padding and setting the tags to <code>display:block</code>, <code>position:static</code>, and <code>width:auto</code> so they just stack up. Seems like a simpler fix and will be much more readable on small screens without squeezing the text into columns.

releasenotes

**Comment by yglazko** [comment]
Followed Craig's suggestion above, added a new commit.

**Comment by craigcook** [comment]
r+! :watermelon: 

Can you squash these two commits and reference the bug number in the message? If your commit message begins with "Fix bug 1189856" our bot will automatically resolve the bug when it merges to master.

**Comment by craigcook** [comment]
Oops, the rebase went a bit pear-shaped. Closing this PR in favor of #3185 from a clean branch.

**Pull request closed without merging**